projects/ccf-rui/src/app/shared/components/block-size-input/block-size-input.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="block-size-input">
+  <div class="block-size-header">
+    Tissue Block Size (mm)
+    <mat-icon class="block-size-refresh" (click)="refreshBlockSize()">refresh</mat-icon>
+  </div>
+  <div class="size-input-form">
+    <div class="form-entry">
+      <mat-form-field>
+        <mat-label class="form-label">Width (X)</mat-label>
+        <input class="form-input" [(value)]="tissueData.width" matInput placeholder="" (input)="updateTissueData($event, 'width')">
+      </mat-form-field>
+    </div>
+    <div class="form-entry">
+      <mat-form-field>
+        <mat-label class="form-label">Height (Z)</mat-label>
```

Comment:
Checking with Leonard, but this really should be Y and not Z. Will ping you when I get an answer from him.

Reply:
So, yes, Height should be (Y) and Depth should be (Z), so please change that.